### DISCUSSION TITLE
Should we handle the Super Simple Export

### DISCUSSION THREAD

**Discussion by miyoku157** [Trouv/bevy_ecs_ldtk#167] [Ideas]
Hi,

I'm wondering if bevy_ecs_ldtk should handle the Super Simple Export format.
https://ldtk.io/docs/game-dev/super-simple-export/
It can sound silly but if someone develop a game with ldtk and bevy_ecs_ldtk and manage to go over all the development process. It could be blocked because he can't release the game without releasing .ldtk file.
 The main problem with that is that anyone can download ldtk and modify any level created by the game creator. 
The export will export different layer as png, which can lead to some performance improved. 
I don't think we shouldn't handle the ldtk format as it is really nice for development purpose. But what about adding this format?

--EDIT--
I rethink about it, and maybe it go against the whole architecture of the bevy_ecs_tilemap used by bevy_ecs_ldtk? Load a png of the whole level could not be optimize against use a tile multiple time with gpu batching?

**Comment by Trouv** [comment]
Yeah it would be pretty different from what the plugin currently does. As you mentioned, `bevy_ecs_tilemap` wouldn't be useful in this scenario, and I'm not sure how we could support LDtk entities or add components to intgrid tiles without the LDtk file. I think it would be a big enough departure from what we do now that it'd be better off as a separate crate. Though, perhaps we could somehow support encrypting LDtk files so that players wouldn't be able to edit them, if that's something enough users care about. Personally I don't mind players editing my levels, but I'm sure others feel differently.
